### PR TITLE
#63 Translate git-novice Section 3

### LINKED ISSUE #63: Translation: git-novice Episode 3
Translation: git-novice Section 3 "Creating a Repository"
-----

* Please assign yourself (or add a comment) to the issue **before** you start translating, so other people know that you are translating.
* Once you have finished translating, send a pull request and reference this issue.

翻訳：git-novice セクション3 "Creating a Repository"
-----

* あなたが翻訳し始めたと分かるように、翻訳を**始める前**にこのイシューを自分に割り振って(assign)、あるいはコメントを書いて下さい。
* 翻訳が終わったら、プルリクエストを開いて、このイシューを参照・リンクして下さい。

https://github.com/swcarpentry-ja/i18n/blob/b3225373bbf9b2f14b8882e4bc019cbd9dbad114/po/git-novice.ja.po#L1533

### PR DESCRIPTION
**Pull request by kozo2**

### CODE REVIEW

File: po/git-novice.ja.po [hunk 1]
```diff
@@ -1556,16 +1556,16 @@ msgid ""
 "Once Git is configured,\n"
 "we can start using it."
 msgstr ""
-"Once Git is configured,\n"
-"we can start using it."
+"Gitが設定されたら、\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"Gitの設定ができたら、\n"
```

File: po/git-novice.ja.po [hunk 2]
```diff
@@ -1588,9 +1588,9 @@ msgid ""
 "reference#repository)—a place where\n"
 "Git can store versions of our files:"
 msgstr ""
-"Then we tell Git to make `planets` a [repository]({{ page.root }}/"
-"reference#repository)—a place where\n"
-"Git can store versions of our files:"
+"次に、Gitに`planets`を[リポジトリ]({{ page.root }}/"
+"reference#repository)—\n"
+"Gitがファイルのバージョンを保存できる場所:にするように伝えます。"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"（Gitがファイルのバージョンを保存できる場所）にするように伝えます。"
```

File: po/git-novice.ja.po [hunk 3]
```diff
@@ -1758,8 +1757,8 @@ msgid ""
 "If you are using a different version of `git`, the exact\n"
 "wording of the output might be slightly different."
 msgstr ""
-"If you are using a different version of `git`, the exact\n"
-"wording of the output might be slightly different."
+"異なるバージョンの `git` を使用している場合、\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"使用している`git`のバージョンによって、\n"
```

File: po/git-novice.ja.po [hunk 4]
```diff
@@ -1783,7 +1782,7 @@ msgstr ""
 # blockquote, which can be cascaded
 #: git-novice/_episodes/03-create.md:80
 msgid "> ## Places to Create Git Repositories"
-msgstr "> ## Places to Create Git Repositories"
+msgstr "> ## Git リポジトリを作るための場所"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
msgstr "> ## Git リポジトリを作る場所"
```

File: po/git-novice.ja.po [hunk 5]
```diff
@@ -1848,54 +1847,46 @@ msgid ""
...
 msgstr ""
 ">\n"
-"> Along with tracking information about planets (the project we have already "
-"created), \n"
-"> のび太 would also like to track information about moons.\n"
-"> Despite ドラえもん's concerns, のび太 creates a `moons` project inside his "
-"`planets` \n"
-"> project with the following sequence of commands:\n"
+"> planets (すでに作成したプロジェクト) "
+"についての追跡情報と共に、 \n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"についての情報を管理すると共に、 \n"
```

File: po/git-novice.ja.po [hunk 6]
```diff
@@ -1848,54 +1847,46 @@ msgid ""
...
+"> のび太 は moons についての情報も追跡したいと考えています。\n"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: We've decided to stick with the original "Dracula" and "Wolfman"
```suggestion
"> ドラキュラは moons についての情報も追跡したいと考えています。\n"
```

File: po/git-novice.ja.po [hunk 7]
```diff
@@ -1848,54 +1847,46 @@ msgid ""
...
+"> ドラえもん の心配にもかかわらず、のび太 は次の一連のコマンドを使って、"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> 狼男の心配にもかかわらず、ドラキュラは次の一連のコマンドを使って、"
```

File: po/git-novice.ja.po [hunk 8]
```diff
@@ -1848,54 +1847,46 @@ msgid ""
...
-"> > because the `planets` repository will track all files, sub-directories, "
-"and \n"
-"> > sub-directory files under the `planets` directory.  Thus, in order to "
-"track \n"
-"> > all information about moons, のび太 only needed to add the `moons` sub-"
-"directory\n"
-"> > to the `planets` directory.\n"
+"> > ## 解答\n"
+"> >\n"
+"> > いいえ。のび太 は `moons` サブディレクトリを Git リポジトリにする"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> > いいえ。ドラキュラは `moons` サブディレクトリを Git リポジトリにする"
```

File: po/git-novice.ja.po [hunk 9]
```diff
@@ -1848,54 +1847,46 @@ msgid ""
...
+"必要はありません。 \n"
+"> > `planets` リポジトリは、`planets` ディレクトリの下のすべてのファイル、サブディレクトリ、"
+"> > およびサブディレクトリファイルを追跡するからです。従って、"
+"> > moons についてのすべての情報を追跡するのは、のび太が `moons` サブディレクトリを"
```
Comment (joelnitta) [inline po/git-novice.ja.po]:
```suggestion
"> > moons についてのすべての情報を追跡するのは、ドラキュラが `moons` サブディレクトリを"
```

**Review by joelnitta — COMMENTED** [review]
Thanks for the PR! I just have a few suggested changes. Please note that we will just use 狼男　and ドラキュラ for "Wolfman" and "Dracula".
Also, can you please translate these parts of the front-matter (lines 1541-1552):
"Creating a Repository"
"Where does Git store information?"
"Create a local Git repository"
"`git init` initializes a repository"
"Git stores all of its repository data in the `.git` directory"

File: po/git-novice.ja.po (hunk 5 above)
Comment (kozo2) [reply po/git-novice.ja.po]:
```suggestion
"についての情報を追跡すると共に、 \n"
```

Comment (kozo2) [reply po/git-novice.ja.po]: I use "追跡する" instead of "管理する".
Because the next translation is "> ドラキュラは moons についての情報も追跡したいと考えています。\n"

**Comment by kozo2** [comment]
@joelnitta 
Thanks for the comment!
I reflected the suggested changes and translated the front-matter.
Could you please check the update?

File: po/git-novice.ja.po [hunk 10]
```diff
@@ -1758,8 +1757,8 @@ msgid ""
 "If you are using a different version of `git`, the exact\n"
 "wording of the output might be slightly different."
 msgstr ""
-"If you are using a different version of `git`, the exact\n"
-"wording of the output might be slightly different."
+"使用している`git`のバージョンによって、\n"
+"出力の正確な表現は少し異なっているかもしれません。"
```
Comment (joelnitta) [inline po/git-novice.ja.po]: In this case, English "exact" means "specific", not "correct". I would remove 正確.
```suggestion
"出力の表現がし異なるかもしれません。"
```

**Comment by joelnitta** [comment]
Looks great, thanks! Please see one additional comment above.
Now we just need one more reviewer to check it.

File: po/git-novice.ja.po (hunk 10 above)
Comment (kozo2) [reply po/git-novice.ja.po]:
```suggestion
"出力の表現が少し異なるかもしれません。"
```

**Review by rikutakei — APPROVED** [review]
Awesome work! I think this is good to go :)

**Review by joelnitta — APPROVED** [review]

**Pull request merged**